Games/League of Legends/BETA Client/script.js

```diff
@@ -1,32 +1,47 @@
 include(["Functions", "QuickScript", "CustomInstallerScript"]);
+include(["Functions", "Shortcuts","Wine"]);
 include(["Functions", "Verbs", "sandbox"]);
+include(["Functions", "Verbs", "d3dx9"]);
 include(["Functions", "Verbs", "vcrun2015"]);
 
 new CustomInstallerScript()
     .name("League of Legends BETA Client")
     .editor("Riot Games")
     .applicationHomepage("http://euw.leagueoflegends.com/")
-    .author("Plata")
+    .author("Plata,feanor12")
     .installationCommand(function(wizard) {
-        return {command: "msiexec", args: ["/i", "C://LoL/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
+        return {command: "msiexec", args: ["/i", "C://LoL_tmp/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
     })
     .category("Games")
     .wineDistribution("staging")
     .wineVersion("2.0")
     .preInstall(function(wine, wizard) {
-        wine.sandbox();
-
         var setupFile = new Resource()
             .wizard(wizard)
             .url("https://riotgamespatcher-a.akamaihd.net/ShellInstaller/EUW/LeagueofLegends_EUW_Installer_2016_11_10.exe")
             .checksum("cbbf9883a59d2dada87e6aa8cc4484b07ea797da")
             .name("LeagueofLegends_EUW_Installer_2016_11_10.exe")
             .get();
 
+        mkdir(wine.prefixDirectory+"drive_c/LoL_tmp");
         mkdir(wine.prefixDirectory+"drive_c/LoL");
-        wine.run(setupFile,["/extract:C:\\LoL","/exenoui"]).wait();
+        var launcher = wine.prefixDirectory+"drive_c/LoL/lol.launcher.admin.exe"
+        var client = wine.prefixDirectory+"drive_c/LoL/LeagueClient.exe"
+        lns(launcher,client)           
+        new WineShortcut()
```

Comment:
You could use writeToFile. But if the launcher does not overwrite an existing LeagueClient.exe, it will not work anyways. I leave it for @qparis to also have a look (maybe he has another idea). Apart from that I think we can merge this.